rtv/content.py

```diff
@@ -105,7 +106,8 @@ def strip_praw_submission(sub):
         displayed through the terminal.
         """
 
-        is_selfpost = lambda s: s.startswith('http://www.reddit.com/r/')
+        reddit_link = re.compile("https?://(www.)?(np.)?redd(it.com|.it)/r/.*")
```

Comment:
I'm pretty sure your periods need to be surrounded with ``[.]``, otherwise they act like wildcards.